lua/starfall/libs_sh/http.lua

```diff
@@ -58,8 +82,8 @@ function http_library.get(url, callbackSuccess, callbackFail, headers)
 		end
 	end
 
-	if requests[instance.player] then SF.Throw("You can't run a new http request yet", 2) end
-	requests[instance.player] = true
+	if not canRequest(instance.player) then SF.Throw(string.format("You have hit the maximum amount of concurrent HTTP requests (%d)", http_max_active:GetInt()), 2) end
```

Comment:
I know it's not a huge deal but I just find it cleaner to use concat instead of string.format. Could you replace with `"You have hit the maximum amount of concurrent HTTP requests ("..http_max_active:GetInt()..")"`